ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -2064,5 +2064,76 @@
       "name": "type-python.svg",
       "mediaType": "image/svg+xml"
     }
-  }
+  },
+  {
+    "id": "vert.x",
+    "creator": "Clement Escoffier",
+    "name": "Eclipse Vert.x",
+    "description": "Eclipse Vert.x Stack on CentOS.",
+    "scope": "advanced",
+    "tags": [
+        "Java",
+        "JDK",
+        "Maven",
+        "Vert.x",
+        "CentOS",
+        "Git"
+    ],
+    "components": [{
+        "name": "CentOS",
+        "version": "---"
+    }, {
+        "name": "JDK",
+        "version": "1.8.0_45"
+    }, {
+        "name": "Maven",
+        "version": "3.2.2"
+    }],
+    "source": {
+        "type": "image",
+        "origin": "registry.centos.org/che-stacks/vertx"
+    },
+    "workspaceConfig": {
+        "environments": {
+            "default": {
+                "machines": {
+                    "dev-machine": {
+                        "agents": [
+                            "org.eclipse.che.terminal",
+                            "org.eclipse.che.ws-agent",
+                            "org.eclipse.che.ssh"
+                        ],
+                        "servers": {},
+                        "attributes": {
+                            "memoryLimitBytes": "2147483648"
+                        }
+                    }
+                },
+                "recipe": {
+                    "location": "registry.centos.org/dharmit/vertx",
```

Comment:
@sbryzak should we replace this with `registry.centos.org/che-stacks/vertx` as well?